### PR TITLE
Issue #66: [Android] add array min function

### PR DESCRIPTION
**Pull request by xstream**

### CODE REVIEW

File: test_data/expression_test_data/function_signatures_array.json [hunk 1]
```diff
@@ -752,6 +752,21 @@
         "android",
         "ios"
       ]
+    },
+    {
+      "name": "min(array) number",
```
Comment (lunarstill) [inline test_data/expression_test_data/function_signatures_array.json:757]: `name` property is not needed.

File: test_data/expression_test_data/function_signatures_array.json [hunk 2]
```diff
@@ -752,6 +752,21 @@
...
+      "function_name": "min",
+      "doc": "Gets the minimun value of array. Returns number value.",
+      "arguments": [
+        {
+          "type": "array",
+          "doc": "Array value to get length of."
```
Comment (lunarstill) [inline test_data/expression_test_data/function_signatures_array.json:763]: Looks like copy-paste.

File: test_data/expression_test_data/functions_array.json
Comment (lunarstill) [inline test_data/expression_test_data/functions_array.json:1]: More tests needed: empty array, non-numeric array.

File: test_data/expression_test_data/functions_array.json [hunk 3]
```diff
@@ -913,6 +913,72 @@
         "android",
         "ios"
       ]
+    },
+    {
+      "name": "min(array) => -1",
```
Comment (lunarstill) [inline test_data/expression_test_data/functions_array.json:918]: `name` property is obsolete, you can should it.

**Comment by lunarstill** [comment]
Nothing happens here.
